prowler/providers/aws/services/elasticache/elasticache_service.py

```diff
@@ -177,3 +180,4 @@ class ReplicationGroup(BaseModel):
     multi_az: str
     tags: Optional[list]
     auto_minor_version_upgrade: bool = False
+    automatic_failover: Optional[str] = None
```

Comment:
You are already setting a default value when creating the model inside the function.
```suggestion
    auto_minor_version_upgrade: bool
    automatic_failover: str
```

Reply:
If I don't set the default value here, several tests for this and other checks will fail as a result. This occurs because when we mock them, the value is never assigned.

Should I leave it as it was, or should I update every mock instance across all checks?